docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,341 @@
+// Issue #480. #434, #463 refer.
+:ap: Asciidoc Python
+:ad: Asciidoctor
+:url-diagram: https://github.com/asciidoc/asciidoc-diagram
+:url-migrate: https://github.com/asciidoc/docs/migration
+:url-recommended: http://asciidoctor.org/docs/asciidoc-recommended-practices
+:url-diffs: http://asciidoctor.org/docs/asciidoc-asciidoctor-diffs/
+== Migrating from {ap} to {ad}
+
+The purpose of this section is to get documents that converted correctly using {ap} to produce the same output using {ad}.
+The differences are very small so that most documents will need very few or no changes.
+Once you have done the basic changes, this section also describes what to change to make use of all the new features in {ad}.
+
+=== Command line
+
+{ad} was designed from the outset to be a drop-in replacement for {ap}. 
+For most documents therefore you can just replace the {ap} call:
+
+`{ad} mydocument.adoc`
+
+with this {ad} call:
+
+`{ad} -a compat-mode mydocument.adoc`
+
+If this doesn't work for you, check out the differences below, particularly <<migrate-deprecated>> and <<migrate-extensions>>.
+
+==== internationalisation
+
+// ref ap UG, not checked
+{ap} has built-in .conf files so that it can translate captions just by setting `-a lang=<language code>`. 
+See <<language-support>> for the {ad} equivalent.
+
+==== Configuration files
+
+{ad} does not use .conf files or filters, so, `--conf-file`, `-dump-conf`, and `--filter` are not implemented. 
+
+==== Themes
+
+// ref ap UG. Actually seems to set JS as well. OK to ignore?
+{ap} themes are just stylesheets, so replace `--theme` with `-a stylesheet=<stylesheet>`.
+
+==== Compact
+
+// ref ap UG. 
+{ap} `--compact` reduces line spacing in lists. 
+To achieve the same in {ad}, use a custom stylesheet.
```

Comment:
AsciiDoc Python never had a compact option. Asciidoctor briefly had a compact option, but it was for compressing whitespace in the generated file. Can you drop this section?